src/main/java/org/apache/pirk/schema/data/partitioner/PrimitiveTypePartitioner.java

```diff
@@ -32,7 +32,7 @@
  * Class for partitioning objects with primitive Java types
  * 
  */
-public class PrimitiveTypePartitioner implements DataPartitioner
+final public class PrimitiveTypePartitioner implements DataPartitioner
```

Comment:
...and I thought I was pedantic ;-)

Yep, will fix that in next round of clean-up over this code --thanks.